store/src/main/java/org/apache/rocketmq/store/stats/BrokerStatsManager.java

```diff
@@ -29,37 +30,37 @@
 
 public class BrokerStatsManager {
 
-    public static final String QUEUE_PUT_NUMS = "QUEUE_PUT_NUMS";
-    public static final String QUEUE_PUT_SIZE = "QUEUE_PUT_SIZE";
-    public static final String QUEUE_GET_NUMS = "QUEUE_GET_NUMS";
-    public static final String QUEUE_GET_SIZE = "QUEUE_GET_SIZE";
-    public static final String TOPIC_PUT_NUMS = "TOPIC_PUT_NUMS";
-    public static final String TOPIC_PUT_SIZE = "TOPIC_PUT_SIZE";
-    public static final String GROUP_GET_NUMS = "GROUP_GET_NUMS";
-    public static final String GROUP_GET_SIZE = "GROUP_GET_SIZE";
-    public static final String SNDBCK_PUT_NUMS = "SNDBCK_PUT_NUMS";
-    public static final String BROKER_PUT_NUMS = "BROKER_PUT_NUMS";
-    public static final String BROKER_GET_NUMS = "BROKER_GET_NUMS";
-    public static final String GROUP_GET_FROM_DISK_NUMS = "GROUP_GET_FROM_DISK_NUMS";
-    public static final String GROUP_GET_FROM_DISK_SIZE = "GROUP_GET_FROM_DISK_SIZE";
-    public static final String BROKER_GET_FROM_DISK_NUMS = "BROKER_GET_FROM_DISK_NUMS";
-    public static final String BROKER_GET_FROM_DISK_SIZE = "BROKER_GET_FROM_DISK_SIZE";
+    @Deprecated public static final String QUEUE_PUT_NUMS = Stats.QUEUE_PUT_NUMS;
```

Comment:
You can move all deprecated constants to `Stats`, and remove `Stats.` in front of all constant names.
IDEA's `Move Members` can do it quickly

Reply:
I am moving these from rocketmq-store to rocketmq-common, so that the user who wants to get statistics dont have to depend on rocketmq-store.